shape.c

```diff
@@ -733,9 +733,10 @@ rb_shape_get_next(rb_shape_t *shape, VALUE obj, ID id)
                 if (RCLASS_EXT(klass)->variation_count >= SHAPE_MAX_VARIATIONS) {
                     rb_category_warn(
                         RB_WARN_CATEGORY_PERFORMANCE,
-                        "Maximum shapes variations (%d) reached by %"PRIsVALUE", instance variables accesses will be slower.",
-                        SHAPE_MAX_VARIATIONS,
-                        rb_class_path(klass)
+                        "The class %"PRIsVALUE" reached %d shape variations, instance variables accesses will be slower amd memory usage increased.\n"
```

Comment:
```suggestion
                        "The class %"PRIsVALUE" reached %d shape variations, instance variables accesses will be slower and memory usage increased.\n"
```

Reply:
Thanks, for my defense I'm not done drinking my ☕  😉